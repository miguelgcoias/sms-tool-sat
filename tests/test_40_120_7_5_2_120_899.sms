40
35 5 45 3 2 1 2
85 7 99 4 2 1 2 2
0 2 4 1 2
39 2 41 1 2
37 5 42 2 1 4
74 6 80 1 6
22 6 28 5 1 1 2 1 1
95 5 105 2 1 4
22 6 28 4 1 2 2 1
23 4 31 3 2 1 1
8 2 10 2 1 1
33 7 40 3 3 1 3
19 6 25 3 1 2 3
48 1 49 1 1
34 6 40 1 6
87 5 97 4 1 1 1 2
91 2 95 1 2
44 2 48 2 1 1
67 1 68 1 1
48 3 51 3 1 1 1
23 5 28 1 5
116 6 122 1 6
73 5 83 1 5
116 2 120 2 1 1
32 6 38 4 1 1 2 2
112 1 113 1 1
23 4 27 4 1 1 1 1
19 4 27 4 1 1 1 1
89 2 93 2 1 1
44 1 46 1 1
88 6 100 5 1 1 1 1 2
17 1 19 1 1
43 3 49 3 1 1 1
79 3 82 2 2 1
110 3 113 2 1 2
3 1 4 1 1
27 4 35 2 2 2
33 6 39 1 6
13 6 19 1 6
81 6 87 5 1 1 2 1 1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
