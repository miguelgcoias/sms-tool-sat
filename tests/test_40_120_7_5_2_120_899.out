22
1 35 37 41
3 0
4 39
8 100 101
10 27 29 30
11 8 9
14 48
16 87 88 93 95
19 67
23 73
24 116 117
26 112
27 23 24 25 26
28 19 20 21 22
29 89 90
30 44
31 91 92 94 97 98
33 45 46 47
34 79 81
36 3
37 31 33
39 13
